app/namespaces/data_provider/data_provider_controller.py

```diff
@@ -179,14 +122,15 @@ def post(self):
         reverse = data.get('reverse', None)
         columns = data.get('columns')
         research_fields = data.get('research_fields')
-        prioritize_estimates = data.get('prioritize_estimates', True)
+        estimates_subgroup = data.get('estimates_subgroup', 'all_estimates')
```

Comment:
Should the default be "all_estimates" or "primary_estimates"? Given that the dashboard is our main client, I think it should be "primary_estimates"